internal/verifier/migration_verifier.go

```diff
@@ -399,8 +399,10 @@ func (verifier *Verifier) getGenerationWhileLocked() (int, bool) {
 
 func (verifier *Verifier) maybeAppendGlobalFilterToPredicates(predicates []bson.D) []bson.D {
 	if verifier.globalFilter == nil {
+		verifier.logger.Debug().Str("filter", fmt.Sprintf("%v", verifier.globalFilter)).Msg("No filter to append")
```

Comment:
If we're in this branch, we know that verifier.globalFilter is nil, and so we also know that `fmt.Sprintf("%v")` is always going to return `<nil>`. I wonder if it would be more useful to not include this Str() call at all, and just have the message say "No filter to append; globalFilter is nil" or something.